Readme.md

```diff
@@ -0,0 +1,494 @@
+
+Another variation of the same could be updating multiple documents with one call and `updateMany` come in handy for such use cases. In our CRUD
+app, we would update `cuisine` all restaurants to your favourite type and `borough` to Brooklyn.
+
+```kotlin
+suspend fun updateMultipleDocuments(db: MongoDatabase) {
+    val collection = db.getCollection<Restaurant>("restaurants")
+    val queryParam = Filters.eq(Restaurant::cuisine.name, "Chinese")
+    val updateParams = Updates.combine(
+        Updates.set(Restaurant::cuisine.name, "Indian"),
+        Updates.set(Restaurant::borough.name, "Brooklyn")
+    )
+
+    collection.updateMany(filter = queryParam, update = updateParams).also {
+        println("Total docs matched ${it.matchedCount} and modified ${it.modifiedCount}")
+    }
+}
+```
+
+In these examples, we used `set` and `combine` with `Updates` but there is much more to explore that allows us to do a more intuitive operation
+like set currentDate or timestamp, increase or decrease the value of the field, etc. You can refer to the
+[docs](https://mongodb.github.io/mongo-java-driver/4.9/apidocs/mongodb-driver-core/com/mongodb/client/model/Updates.html) for the complete list.
+
+---------------------
+
+## Delete
+
+Now let's see the last operation of our CRUD app i.e. Delete and start with how to delete a single document and here we would be
+using `findOneAndDelete` instead `deleteOne` which also returns the deleted document as output.
+
+```kotlin
+val collection = db.getCollection<Restaurant>(collectionName = "restaurants")
+val queryParams = Filters.eq("restaurant_id", "restaurantId")
+
+collection.findOneAndDelete(filter = queryParams).also {
+    it?.let {
+        println(it)
+    }
+}
+```
+
+![delete output](https://images.contentstack.io/v3/assets/blt39790b633ee0d5a7/blta4bb9c39c2356306/6489bf30352ac64eebda33c6/Screenshot_2023-06-14_at_14.21.37.png)
+
+And to delete multiple documents we can use `deleteMany`, this would be handy to delete all the data we have created earlier with Create operation.
+
+```kotlin
+suspend fun deleteRestaurants(db: MongoDatabase) {
+    val collection = db.getCollection<Restaurant>(collectionName = "restaurants")
+
+    val queryParams = Filters.or(
+        listOf(
+            Filters.regex(Restaurant::name.name, Pattern.compile("^Insert")),
+            Filters.regex("restaurant_id", Pattern.compile("^restaurant"))
+        )
+    )
+    collection.deleteMany(filter = queryParams).also {
+        println("Document deleted : ${it.deletedCount}")
+    }
+}
+```
+
+---------------------
+
+## Summary
+
+With this blog post, we have covered all the basic operations and have also seen how easily we can leverage the simplicity of Kotlin with the MongoDB
+Kotlin driver. Thank you for reading and hopefully you find this article informative! The complete source code of the app can be found on
+[GitHub][5].
```

Comment:
Do you want to hint at the next tutorials in the series to come?

Reply:
As discussed earlier, this is not a series.